src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
+const indexOfDeadAnimal = function(animalToRemove) {
```

Comment:
is there any reason to assign a function declaration to the variable? It can be stored just as a variable, it seems to me